src/vocgui/models.py

```diff
@@ -28,7 +28,9 @@ class Document(models.Model):  # pylint: disable=R0903
     Contains words + images and relates to a training set
     """
     word = models.CharField(max_length=255)
-    article = models.CharField(max_length=255, choices=[('der', 'der'), ('das', 'das'), ('die', 'die'), ('die (Plural)', 'die (Plural)')], default='')
+    article = models.CharField(max_length=255, choices=[('der', 'der'), ('das', 'das'), 
```

Comment:
The `choices` list should be moved into a module static and then be reused here and for the alternative word as well.

Reply:
alright